databaseconnect.py

```diff
@@ -11,19 +11,32 @@ def connection_to_database():
     import config
     import mysql.connector
     from time import sleep
+    import sys
+
+    called_from = sys.argv[0]
+    logging.debug(called_from)
```

Comment:
`sys.argv[0]` will provide a path that ends with "`pytest`" when called during testing with `pytest`.